src/sentry/api/endpoints/organization_events.py

```diff
@@ -56,6 +57,7 @@ def get(self, request: Request, organization) -> Response:
             "organizations:performance-use-metrics", organization=organization, actor=request.user
         )
         metrics_enhanced = request.GET.get("metricsEnhanced") == "1" and performance_use_metrics
+        sentry_sdk.set_tag("performance.use_metrics", metrics_enhanced)
```

Comment:
we should really just parrot the param name here to avoid confusion, imo `use_metrics` -> `metrics_enhanced`